docs/admin.md

```diff
@@ -0,0 +1,90 @@
+## Admin Manual for Definitely Typed
+
+Welcome! This is a resource for the person who is on call for DefinitelyTyped. The TypeScript team always has someone 
+assigned to DefinitelyTyped duty, where they do a week on-call. You can see the [schedule here](http://aka.ms/DTRotation).
+
+When on-call, your goal is to try keep on top of the many open PRs for DefinitelyTyped; they are categorized into 
+different sections inside the [Projects board](https://github.com/DefinitelyTyped/DefinitelyTyped/projects/4) on this repo.
+
+You should scan from left to right through the boards, and ideally try to start at the oldest PR in a section and work 
+your way through to the newest. There is a tool: [`focus-dt`](https://github.com/DefinitelyTyped/focus-dt) which can help with this.
+
+Our goals are to provide a stable, high quality ecosystem of type definitions and that requires the compiler team 
+making sure that a PR does not break the ecosystem. This work isn't a code review in the usual sense, but an assessment
+of how well the changes fit the rest of the type definitions and if it could break builds for our users or our systems. 
+With the impact understood, your job is to delegate to the people who actually know how to review the code: users 
+of the library who have agreed to look after the types. 
+
+### Looking at a PR
+
+For a [quick TLDR overview, you can read these slides](https://docs.google.com/presentation/d/1Q4xfZSY7d9yHhtxSyb-DE85fTXB38RyF3nnyVyvenwc/edit#slide=id.p).
+
+Some key concepts:
+
+- Popular packages can break more installs, and will need more time and focus
+- There are a handful of authors who have shipped a lot of high quality contributions who you can happy delegate to
+
+##### Amending an existing DefinitelyTyped Package
+
+An ideal PR to a DT package looks like:
+
+- A link in the PR description to the API being added
+- Only additions to the existing types
+- Test code which covers the existing use case
+
+Most of the PRs are like this, in which case then a review for that PR should be pretty quick. Look through the code 
+changes, then see if there are comments asking for the merge to be delayed. If not, then you're good to merge. You 
+can then leave a thanks comment and hit the merge button.
+
+Constraints which you should consider:
+
+- Is it popular? (e.g. do you recognise it) if so, it should probably have two sign-offs
+- Will the PR break existing code? If so, check that there's a semver bump
```

Comment:
This is hard in two ways:

1. Anything that's not a strict addition is likely to break code; and some additions break code, for example if they're to a class with subclasses.
2. DT packages are supposed to match the source's major.minor, so to bump semver, breaking changes can only ship when updating the types to a new major version.

The result is that so far we've told people that the Right Way to consume DT packages is to pin them to an exact major.minor.patch. That's not in the README and it's not common practise, but I think those are the things to change.